tests/models/plbart/test_tokenization_plbart.py

```diff
@@ -129,7 +129,15 @@ def test_full_base_tokenizer(self):
         end = tokenizer.vocab_size
         language_tokens = [tokenizer.convert_ids_to_tokens(x) for x in range(end - 4, end)]
 
-        self.assertListEqual(language_tokens, ["java", "python", "en_XX", "<mask>"])
+        self.assertListEqual(language_tokens, ["__java__", "__python__", "__en_XX__", "<mask>"])
+
+        code = "java.lang.Exception, python.lang.Exception, javascript, php, ruby, go"
+        self.assertEqual(
+            tokenizer.decode(
+                tokenizer([code])["input_ids"][0], skip_special_tokens=True, clean_up_tokenization_spaces=False
+            ),
+            code,
+        )
```

Comment:
```suggestion
        code = "java.lang.Exception, python.lang.Exception, javascript, php, ruby, go"
        input_ids =  tokenizer(code).input_ids
        self.assertEqual(tokenizer.decode(input_ids skip_special_tokens=True, clean_up_tokenization_spaces=False),code)
```
Something like this would be more readable but it is only a small nit. Same for the other example.